### ISSUE TITLE
Display Empty State Message When No Facility Group is Associated with Store Pickup Group Type

### ISSUE THREAD

**Issue by Adameddy26** [hotwax/available-to-promise#354]
## Description
Currently, when no facility group is associated with the store pickup group type, the facilities are still visible in the facilities section, but the toggle button to enable/disable store pickup for those facilities is not present. This can cause confusion for users. To improve user experience, we should implement an enhancement where no facilities are shown in such cases, and an empty state message is displayed to notify the user.

**Comment by dongreparam** [comment]
Upon thorough review, I have determined that the message "**No store pickup group found linked with current product store**" is more appropriate than simply displaying facilities without a toggle button. This message clearly communicates to the user (retailer) that there is no association between the facility group pickup and the current product store, providing clarity and preventing potential confusion.

Upon further inspection, I identified two view entities responsible for retrieving the relevant data: **ProductStoreFacilityDetail** and **ProductStoreAndFacilityGroup**.

The _ProductStoreFacilityDetail_ view entity ensures that facility details linked to the current product store are fetched. Meanwhile, _ProductStoreAndFacilityGroup_ is responsible for linking the product store to the facility group.

In scenarios where a card of facilities is displayed but no toggle is available, it indicates that **ProductStoreAndFacilityGroup** has not retrieved any data. This implies that the current product store is not associated with any facility group pickup, which further justifies the display of the appropriate message.
